classes/data-stores/ActionScheduler_HybridStore.php

```diff
@@ -85,6 +85,24 @@ public function set_autoincrement( $table_name, $table_suffix ) {
 					'status'    => '',
 				]
 			);
+			// Verify the correct action_id was set.
+			$table_name = $wpdb->{ActionScheduler_StoreSchema::ACTIONS_TABLE};
+			$action_id = $wpdb->get_var( "SELECT MAX(action_id) FROM {$table_name}" ); // phpcs:ignore WordPress.DB.PreparedSQL.InterpolatedNotPrepared
+			// If it's lower use an update to set it.
+			if ( $action_id < $this->demarkation_id ) {
```

Comment:
This issue could occur due to `$this->demarkation_id` being `0`, or something else less than the actual `MAX(ID) from wp_posts` value. In that case, even with this change, the issue would persist.

Reply:
Since we don't have any data to go on it's 50/50 that the ID was 0 or there was an issue with inserting the identity. I think what I will look at is adding a check for the ID > 2 or block the migration to this PR.

Reply:
> what I will look at is adding a check for the ID > 2 or block the migration to this PR.

That's a good approach. 👍